main.py

```diff
@@ -202,11 +202,11 @@ def gameLoop(speed):
         # Update round points
         if score1 == const.SCORELIMIT:
             rounds_p1 += 1
-            pygame.mixer.Sound.play(background_music,-1)
+            # pygame.mixer.Sound.play(background_music,-1)

```

Comment:
Please remove the line instead of commenting it. 
If you can try to rewind the background music at start of each round.